lib/src/assets/img/logo-nexus-generic.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Group 7 Copy</title>
```

Comment:
If we're going to include a title in the SVG, it should be a meaningful one.

Reply:
> If we're going to include a title in the SVG, it should be a meaningful one.

SVG extensively cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/e6ac7a8ea2c55cc21584f554f7ae76b798036799